pkg/api/steve/supportconfigs/generator.go

```diff
@@ -136,7 +136,8 @@ func (h *GeneratorHandler) generateSupportConfig() (io.Reader, error) {
 		}
 	}
 
-	return &buf, nil
+	err = tw.Close()
+	return &buf, err
```

Comment:
nit: 
```suggestion
	return &buf, tw.Close()
```

Reply:
I did think about doing this and this was what I originally had. However, I did not like the idea of having a return which called another function and returned a pre-computed value. I'm ok with one function call serving as the return, but one call and one static value just felt off to me. Let me know if you disagree - definitely see the point of having less LOC.

Reply:
I think it's readable either way,  Which is the only argument to be made here. It's purely a style decision at this point, so I would leave it if you prefer this way. One argument being a static value, should be fine and is similar to what is done [in the bytes package](https://cs.opensource.google/go/go/+/refs/tags/go1.18.3:src/bytes/buffer.go;l=168).